packages/alfa-rules/test/sia-r83/rule.spec.tsx

```diff
@@ -503,81 +502,109 @@ test(`evaluate() fails a relatively positioned node clipped by a static parent`,
   ]);
 });
 
-test(`evaluate() passes a relatively positioned node with a handling static parent`, async (t) => {
-  const target = h.text("Hello World");
-
+test("evaluate() is inapplicable to a text node that is not visible", async (t) => {
   const document = h.document(
     [
       <body>
-        <div class="relative clipping">
-          <div class="handle">
-            <span class="relative content">{target}</span>
-          </div>
-        </div>
+        <div hidden>Hello world</div>
       </body>,
     ],
     [
       h.sheet([
-        h.rule.style(".relative", { position: "relative" }),
-        h.rule.style(".clipping", { overflowX: "hidden" }),
-        h.rule.style(".handle", { overflowX: "scroll" }),
-        h.rule.style(".content", { whiteSpace: "nowrap" }),
+        h.rule.style("div", {
+          overflow: "hidden",
+          whiteSpace: "nowrap",
+        }),
       ]),
     ]
   );
 
-  t.deepEqual(await evaluate(R83, { document }), [
-    passed(R83, target, { 1: Outcomes.WrapsText }),
-  ]);
+  t.deepEqual(await evaluate(R83, { document }), [inapplicable(R83)]);
 });
 
-test(`evaluates() passes a clipping element with font-relative height`, async (t) => {
-  const target = h.text("Hello World");
-
+test(`evaluate() is inapplicable to a text node that is excluded from the
+      accessibility tree using the \`aria-hidden\` attribute`, async (t) => {
   const document = h.document(
     [
       <body>
-        <div class="growing">{target}</div>
+        <div aria-hidden="true">Hello world</div>
       </body>,
     ],
     [
       h.sheet([
-        h.rule.style(".growing", {
-          overflowY: "hidden",
-          minHeight: "1.5em",
-          height: "10px",
+        h.rule.style("div", {
+          overflow: "hidden",
+          whiteSpace: "nowrap",
         }),
       ]),
     ]
   );
 
-  t.deepEqual(await evaluate(R83, { document }), [
-    passed(R83, target, { 1: Outcomes.WrapsText }),
-  ]);
+  t.deepEqual(await evaluate(R83, { document }), [inapplicable(R83)]);
 });
 
-test(`evaluates() passes a clipping element with font-relative width`, async (t) => {
-  const target = h.text("Hello World");
+test(`evaluate() ignores text in <option> into a single line <select> element`, async (t) => {
```

Comment:
Actually, we should have test with multi-line select (one with `size`, one with `multiple`), to make sure that `hasDisplaySize` is correct.